build.gradle

```diff
@@ -441,6 +441,44 @@ task bundle(type: (isWindows()) ? Zip : Tar) {
     }
 }
 
+task bundleRestricted(type: Tar) {
+    dependsOn('crossDistPlatformLibs')
```

Comment:
This task and the `bundle` task have a lot of common settings. I'm not sure but probably it worth moving this common part to a single place.  E.g. 
```
task bundle (...) {
    // Bundle-specific configuration
}

task bundleRestricted(...) {
    // Restricted bundle-specific configuration.
}

configure([bundle, bundleRestricted]) {
    // Common configuration.
}
```

In this case we get rid of settings duplication but the bundle configuration becomes divided into two parts. So I'm not sure if it is better or not. What do you think?

Reply:
Sounds great in scenario of task duplication :). I've just forgotten about this fabulous `configure` 😄